alica_engine/src/engine/RunningPlan.cpp

```diff
@@ -339,7 +337,6 @@ void RunningPlan::useEntryPoint(const EntryPoint* value)
 void RunningPlan::useState(const State* s)
 {
     if (_activeTriple.state != s) {
-        ALICA_ASSERT(s == nullptr || (_activeTriple.entryPoint && _activeTriple.entryPoint->isStateReachable(s)));
```

Comment:
I don't think that's what what the original macro did, though.
ALICA_ASSERT(CHECK) (LIKELY(CHECK) ? void(0) : [] { assert(!#CHECK); }()) is the definition - if the check isn't true, then it asserts !check, so as far as I can tell the macro never really did anything?

Reply:
That means logic wise it is the same as a normal assert right?

Reply:
Logic wise I think I think its like 
if cond { void(0) } (no-op)
else { assert(!cond) }
but to make it in to the else block the condition has to be false, so the assert is pointless.

If we replace it with a regular assert, then if the condition is false, it would terminate, but the macro would just assert (!false) and do nothing